src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -119,7 +119,7 @@ class AuthScreens extends React.Component {
         PersonalDetails.fetch();
         User.getUserDetails();
         User.getBetas();
-        fetchAllReports(true, true);
+        fetchAllReports(true, true, 8000);
```

Comment:
This number tests well now, but we might need to revisit in the future, perhaps a constant could be used to provide a bit more context?

Reply:
+1

Reply:
Yeah, I sort of just hate that this is an arbitrary delay, but not sure what else to do. We could separate the fetching of history from `fetchAllReports` and then call it on it's own based on some other event.... but still not quite sure what that event would be.

Rather than make a constant I'll bake this into the method and add more context there since I feel like there's a chance this will change in the future and isn't really the best solution - but works for today.